mobile/src/main/java/ch/epfl/sdp/mobile/application/social/SocialFacade.kt

```diff
@@ -17,18 +19,15 @@ import kotlinx.coroutines.flow.map
  */
 class SocialFacade(private val auth: Auth, private val store: Store) {
 
+  /**
+   * Searches user by exact max on name
```

Comment:
Typo : I think you meant match here ?

```suggestion
   * Searches user by exact match on name
```